webapp/utils/syntax_highlighting.jsx

```diff
@@ -63,62 +63,62 @@ import hljsVerilog from 'highlight.js/lib/languages/verilog.js';
 import hljsXml from 'highlight.js/lib/languages/xml.js';
 import hljsYaml from 'highlight.js/lib/languages/yaml.js';
 
-hlJS.registerLanguage('actionscript', hljsActionscript);
+registerLanguageNames(['actionscript', 'as', 'as3'], hljsActionscript);
```

Comment:
Hey @jazzzz, I pulled your branch locally and it seems like only the original identifiers work.

For example
<img width="202" alt="screen shot 2017-01-11 at 2 49 02 pm" src="https://cloud.githubusercontent.com/assets/2672098/21863873/2b5ebcbe-d80d-11e6-9535-1bbc25514cb6.png">
works, but
<img width="179" alt="screen shot 2017-01-11 at 2 49 19 pm" src="https://cloud.githubusercontent.com/assets/2672098/21863878/33ce6084-d80d-11e6-946b-704b65b4b516.png">
doesn't.

I think it has something to do with `HighlightedLanguages` in `webapp/utils/constants.jsx` if you want to take a look